tools/raml2html/composer.json

```diff
@@ -33,7 +34,19 @@
             "EzSystems\\Raml2Html\\": "src"
         }
     },
+    "extra": {
+        "patches": {
+            "raml-org/raml-php-parser": {
+                "Fix warnings and depreciations for PHP 7.4": "patches/php_7.4.patch"
```

Comment:
do you intend to keep this in the upstream or is this temporary change?

Reply:
@alongosz  I'll make a PR on https://github.com/raml-org/raml-php-parser/blob/master/src/Body.php#L231 at some point but this is a "middle term" projection. So, this is "temporary" for a while.

Reply:
> @alongosz I'll make a PR on https://github.com/raml-org/raml-php-parser/blob/master/src/Body.php#L231 at some point but this is a "middle term" projection. So, this is "temporary" for a while.

TBH I'm not sure if we should keep this in the upstream at any point. Seems rather unstable and indicates that there are possibly more issues with PHP 7.4 support for that 3rd party.
POV ping @ezsystems/php-dev-team

Reply:
@alongosz So, I studied it a little bit deeper and it was fixable on our side: see b79a57bac62578762a0badc36825b960087e28f9
Sorry for the time lost.

Reply:
No worries, great find :muscle: